rcos_io/db.py

```diff
@@ -238,3 +260,25 @@ def get_meetings() -> List[Dict[str, Any]]:
     )
     result = client.execute(query)
     return result["meetings"]
+
+def add_project_lead(project_id: str, user_id: str, semester_id: str, credits: int):
```

Comment:
In the future, we should make this an upsert instead of an insert, but for now this works great!